packages/playground/website/playwright/playwright.ci.config.ts

```diff
@@ -0,0 +1,73 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+	testDir: './e2e',
+	/* Run tests in files in parallel */
+	fullyParallel: true,
+	/* Fail the build on CI if you accidentally left test.only in the source code. */
+	forbidOnly: !!process.env.CI,
+	/* Retry on CI only */
+	retries: process.env.CI ? 2 : 0,
+	/* Opt out of parallel tests on CI. */
+	workers: process.env.CI ? 1 : undefined,
+	/* Reporter to use. See https://playwright.dev/docs/test-reporters */
+	reporter: 'html',
+	/* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+	use: {
+		/* Base URL to use in actions like `await page.goto('/')`. */
+		baseURL: 'http://127.0.0.1/',
+
+		/* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+		trace: 'on-first-retry',
+	},
+
+	timeout: 300000,
+	expect: { timeout: 15000 },
+
+	/* Configure projects for major browsers */
+	projects: [
+		{
+			name: 'chromium',
+			use: { ...devices['Desktop Chrome'] },
+		},
+
+		// {
+		//   name: 'firefox',
+		//   use: { ...devices['Desktop Firefox'] },
+		// },
+
+		// {
+		//   name: 'webkit',
+		//   use: { ...devices['Desktop Safari'] },
+		// },
+
+		/* Test against mobile viewports. */
+		// {
+		//   name: 'Mobile Chrome',
+		//   use: { ...devices['Pixel 5'] },
+		// },
+		// {
+		//   name: 'Mobile Safari',
+		//   use: { ...devices['iPhone 12'] },
+		// },
+
+		/* Test against branded browsers. */
+		// {
+		//   name: 'Microsoft Edge',
+		//   use: { ...devices['Desktop Edge'], channel: 'msedge' },
+		// },
+		// {
+		//   name: 'Google Chrome',
+		//   use: { ...devices['Desktop Chrome'], channel: 'chrome' },
+		// },
```

Comment:
Should we enable more browsers in CI?

Reply:
Testing on Safari and mobile safari would be a **really** good addition, especially once we have offline mode testing. Let's track it in a separate issue/PR. Btw, for testing the offline mode, we should just kill the server after the first page load instead of trusting the browser's "offline mode" feature.

Reply:
Enabling more browsers would be great, but I agree it would be better in a separate PR.

I also wonder whether more thorough E2E testing (more browsers and maybe more tests) could be done less frequently via a separate CI workflow.

Reply:
New issue https://github.com/WordPress/wordpress-playground/issues/1804

> Let's track it in a separate issue/PR. Btw, for testing the offline mode, we should just kill the server after the first page load instead of trusting the browser's "offline mode" feature.

I agree, the browser offline feature isn't great.